client/state/selectors/is-gutenframe-enabled.js

```diff
@@ -0,0 +1,55 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+import { isHttps } from 'lib/url';
+import versionCompare from 'lib/version-compare';
+import getWordPressVersion from 'state/selectors/get-wordpress-version';
+import isPluginActive from 'state/selectors/is-plugin-active';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import isVipSite from 'state/selectors/is-vip-site';
+import { getSiteAdminUrl, isJetpackMinimumVersion, isJetpackSite } from 'state/sites/selectors';
+
+export const isGutenframeEnabled = ( state, siteId ) => {
```

Comment:
Do we really need to separate this out? I think it would be easier to grok if we only had one function going forward - `isGutenbergEnabled` (or `isGutenframeEnabled` if you prefer that name).

To clarify - we shouldn't have the need to call `isGutenframeEnabled` separately anywhere in the app, with unified approach `isGutenbergEnabled` should cover all the cases.